hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/AwsAddressTranslator.java

```diff
@@ -35,7 +36,7 @@
 public class AwsAddressTranslator implements AddressTranslator {
 
     private final ILogger logger;
-    private volatile Map<String, String> privateToPublic;
+    private volatile Map<String, String> privateToPublic = new HashMap<String, String>();
```

Comment:
do we still need following checking after this initialization?
`return table != null ? table : Collections.<String, String>emptyMap();`